examples/4-tests/example_test.go

```diff
@@ -353,10 +353,16 @@ func Test_7InputOutputWithHeaders(t *testing.T) {
 	proc, _ := goka.NewProcessor([]string{}, goka.DefineGroup("group",
 		goka.Input("input", new(codec.String), func(ctx goka.Context, msg interface{}) {
 			processorHeaders = ctx.Headers()
-			ctx.Emit("output", ctx.Key(), fmt.Sprintf("forwarded: %v", msg), goka.WithCtxEmitHeaders(
-				map[string][]byte{
-					"Header1": []byte("to output"),
-				}))
+			ctx.Emit("output", ctx.Key(), fmt.Sprintf("forwarded: %v", msg),
+				goka.WithCtxEmitHeaders(
+					map[string][]byte{
```

Comment:
Would be great to add colliding headers as well to assert overwrite behaviour, but not critical.

Reply:
No problem, just added it.